lib/aerospike/client.rb

```diff
@@ -314,7 +314,7 @@ def get_header(key, options={})
     #  The returned records are in positional order with the original key array order.
     #  If a key is not found, the positional record will be nil.
     #  The policy can be used to specify timeouts.
-    def batch_get(keys, bin_names=[], options={})
+    def batch_get(keys, bin_names = nil, options = nil)
```

Comment:
Changing the default value for `bin_names` from `[]` to `nil` could potentially break an application that calls `Client#batch_get` without passing `bin_names` since we call `bin_names.uniq` further down when creating the `BatchCommandGet` instances.

Reply:
Missed that (and apparently specs did too). Imho `uniq` is not necessary here, and protecting from duplicate bins seems a bit overkill. What do you think?

Reply:
Agree, let's remove the `uniq` call. Not sure why it was added in the first place. None of the other commands that take a list of bin names as parameter do that, I think.

I'm afraid the specs don't cover a lot of the edge cases so I'm not surprised they didn't catch this.